ipaserver/install/httpinstance.py

```diff
@@ -613,9 +613,20 @@ def migrate_to_mod_ssl(self):
             certs.install_pem_from_p12(temp.name,
                                        pk12_password,
                                        paths.HTTPD_CERT_FILE)
+            with open(
+                os.path.join(
+                    paths.IPA_PASSWD_DIR,
+                    HTTPD_PASSWD_FILE_FMT.format(host=api.env.host)
+                    ), 'wb') as passwd_file:
+                os.fchmod(passwd_file.fileno(), 0o600)
+                passwd_fname = passwd_file.name
```

Comment:
This looks backward. How about you set ``passwd_fname`` first and then use it in ``with open()`` and ``certs.install_key_from_p12()``.

Reply:
Yeah, I noticed that, too, I wrote this way too fast to think about such things :) Will fix.